rust/config/mainnet3_config.json

```diff
@@ -432,6 +432,10 @@
       "grpcUrl": "https://grpc-kralum.neutron-1.neutron.org:80",
       "canonicalAsset": "untrn",
       "prefix": "neutron",
+      "minimumGasPrice": {
+        "amount": "1",
```

Comment:
Should probably set this to 0.5 https://github.com/cosmos/chain-registry/blob/master/neutron/chain.json#L20